src/erlcloud_ddb2.erl

```diff
@@ -2577,6 +2579,28 @@ list_tables(Opts, Config) ->
     out(Return, fun(Json, UOpts) -> undynamize_record(list_tables_record(), Json, UOpts) end, 
         DdbOpts, #ddb2_list_tables.table_names, {ok, []}).
 
+
+list_all_tables() ->
```

Comment:
I should also note that I think this should be called `list_tables_all` to be in keeping with the overall style.

Reply:
OK! : )